subscriptions/client/database/subscriptions_test.go

```diff
@@ -0,0 +1,116 @@
+package database
+
+import (
+	"database/sql"
+	"fmt"
+	"main/config"
+	"main/models"
+	"testing"
+
+	_ "github.com/lib/pq"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetSubscriptions(t *testing.T) {
+	db, dbClose := DbHelper(t)
+	defer dbClose()
+
+	SetUp(t, db, nil)
+	t.Cleanup(func() {
+	})
+
+	t.Run("succes", func(t *testing.T) {
+		s := Client{DB: db}
+
+		subscriptions, err := s.GetSubscriptions()
+		require.NoError(t, err)
+
+		dbSubscriptions := []models.UserSubscription{}
+		rows, err := db.Query("SELECT user_id, subscription_id, status, charge_amount FROM subscriptions")
+		require.NoError(t, err)
+
+		for rows.Next() {
+			dbSubscription := models.UserSubscription{}
+			err := rows.Scan(&dbSubscription.UserID, &dbSubscription.SubscriptionID, &dbSubscription.Status, &dbSubscription.ChargeAmount)
+			require.NoError(t, err)
+			dbSubscriptions = append(dbSubscriptions, dbSubscription)
+		}
+		assert.Equal(t, subscriptions, dbSubscriptions)
+	})
+
+}
+
+func TestGetUserSubscriptions(t *testing.T) {
+	test := []struct {
+		name          string
+		userID        string
+		subscriptions []models.UserSubscription
+		err           error
+	}{
+		{
+			name:   "success",
+			userID: "1",
+			err:    nil,
```

Comment:
probably success case receiving nothing from the DB as well. Consider put something there before fetching